### PR TITLE
Add PPID to the stats

### PR DESCRIPTION
**Pull request by simonepri**
## Changes
- Add `ppid` property to the stats object
- Remove unnecessary filter.
- Fix wrong dependency name

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
+    var pids = children.map(function (p) {
       return p.PID
     })
 
-    pidusage.stat(children, cb)
+    pidusage.stat(pids, function(err, stats) {
+      if (err) return cb(err)
+
+      stats.forEach(function (s, i) {
+        if (!s) return
```
Comment (soyuka) [inline index.js:16]: how is this possible?

Comment (simonepri) [reply index.js:16]: If the provided pid hasn't been read.

**Comment by soyuka** [comment]
thanks @simonepri !

**Comment by simonepri** [comment]
Anyway there's something wrong with `pidusage`
I don't think it is ready to be published.
Tests aren't complete enough.
image

@soyuka

**Comment by soyuka** [comment]
Interesting issue how may I reproduce? Also there's a bug when you give a pid that doesn't exist `ps` gives a blank line (on it right now).

**Pull request merged**